cli-commands/kc.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+UbiCli.base("kc") do
+  banner "ubi kc command [...]"
+  post_banner "ubi kc (location/cluster-name | kc-id) post-command [...]"
```

Comment:
```suggestion
  post_banner "ubi kc (location/kc-name | kc-id) post-command [...]"
```

For consistency with the other commands.